app/src/main/java/ro/code4/monitorizarevot/ui/main/MainActivity.kt

```diff
@@ -70,9 +78,19 @@ class MainActivity : BaseActivity<MainViewModel>() {
 
     }
 
-
     override fun onSupportNavigateUp(): Boolean {
         return navController.navigateUp(appBarConfiguration) || super.onSupportNavigateUp()
     }
 
+    private fun logoutUser() {
+        sharedPreferences.deleteToken()
+        openLogin()
+    }
+
+    private fun openLogin() {
+        val intent = Intent(this, LoginActivity::class.java)
+        startActivity(intent)
+        finish()
```

Comment:
You could call `finishAffinity()` in order to make sure there are no activities in the backstack